src/field/tv.c

```diff
@@ -105,6 +86,351 @@ static u8 gUnknown_03000720;
 static u8 gUnknown_03000721;
 static s8 gUnknown_03000722;
 
+const struct OutbreakPokemon gPokeOutbreakSpeciesList[5] =
```

Comment:
I believe we want to prefix static rodata with "s", rather than "g".  We should address that later, though.